lib/include/openamp/rpmsg_virtio.h

```diff
@@ -165,6 +165,12 @@ rpmsg_virtio_create_virtqueues(struct rpmsg_virtio_device *rvdev,
 					callbacks, NULL);
 }
 
+static inline void
+rpmsg_virtio_delete_virtqueues(struct rpmsg_virtio_device *rvdev)
+{
+	virtio_delete_virtqueues(rvdev->vdev);
```

Comment:
virtio_delete_virtqueues return a value that is just ignored here.

Reply:
Has done in https://github.com/OpenAMP/open-amp/pull/527, let's wait this PR merged and rebase to last.